package-lock.json

```diff
@@ -34862,26 +34862,26 @@
         "@rollup/plugin-node-resolve": "^15.0.1",
         "@rollup/plugin-replace": "^5.0.1",
         "@rollup/plugin-terser": "^0.4.0",
-        "@storybook/addon-a11y": "^7.4.5",
-        "@storybook/addon-actions": "^7.4.5",
-        "@storybook/addon-docs": "^7.4.5",
-        "@storybook/addon-essentials": "^7.4.5",
-        "@storybook/addon-interactions": "^7.4.5",
-        "@storybook/addon-links": "^7.4.5",
+        "@storybook/addon-a11y": "^7.4.0",
+        "@storybook/addon-actions": "^7.4.0",
+        "@storybook/addon-docs": "^7.0.4",
```

Comment:
What's going on here? Why is addon-docs being downgraded?

Reply:
Oh this section is just reflecting the version ranges expressed in the package.json today: https://github.com/ni/nimble/blob/2c1247b7a9c46d7e10b36b9a53f3cff346e8d0d5/packages/nimble-components/package.json#L107

So it's not really a downgrade, the resolved versions are staying high, this is just duplicated metadata.